src/ln/functional_test_utils.rs

```diff
@@ -805,7 +805,7 @@ pub fn fail_payment(origin_node: &Node, expected_route: &[&Node], our_payment_ha
 	fail_payment_along_route(origin_node, expected_route, false, our_payment_hash);
 }
 
-pub fn create_network(node_count: usize) -> Vec<Node> {
+pub fn create_network(node_count: usize, node_config: Vec<Option<UserConfig>>) -> Vec<Node> {
```

Comment:
Just use a &[] unless there is a real need for a Vec, this avoids a bunch of heap allocation overhead.

Reply:
Done